slate-social-media/src/Components/Post Model/PostModel.js

```diff
@@ -82,45 +98,74 @@ function PostModel({ postdata }) {
 
         <div class="reference">
           <div class="reference-content">
-            {image && (
+            {image ? (
               <img class="reference-thumb" src={image} alt="uploaded-by-user" />
+            ) : (
+              <div>{null} </div>
+            )}
+            {video ? (
+              <video class="reference-video" src={video} controls />
+            ) : (
+              <div>{null} </div>
+            )}
+
+            {pdf ? (
+              <embed class="reference-video" src={pdf} />
+            ) : (
+              <div> {null}</div>
             )}
-            {video && <video class="reference-video" src={video} controls />}
           </div>
         </div>
         <div class="social">
           <div class="social-buttons">
+            <div style={{ color: "red", marginRight: "-5rem" }}>
+              {likeCount}
+            </div>
             {getUserDetails?.username === isLiked?.username &&
             getUserDetails?._id ? (
               <span
                 class="material-icons postcardmi"
-                onClick={() => dislikesPostFn(postDispatch, _id)}
+                onClick={() => dispatch(dislikesPostFn(_id))}
               >
                 favorite
               </span>
             ) : (
               <span
                 class="material-icons postcardmi"
-                onClick={() => likesPostFn(postDispatch, _id)}
+                onClick={() => dispatch(likesPostFn(_id))}
               >
                 favorite_border
               </span>
             )}
-            {likeCount}
+
             <Link to={`/comments/${_id}`}>
-              <span class="material-icons postcardmi">chat_bubble_outline</span>
+              <div
```

Comment:
Avoid inline styling

Reply:
thanks for all the suggestions and code review I am looking forward to implement all the suggestions